scripts/deploy

```diff
@@ -235,6 +235,63 @@ EOF
   fi
 }
 
+create_restore() {
+  KUBECTL_CMD="kubectl"
+  insecure=false
+  case $# in
+    1) ;;
+    2) ;;
+    *) echo "Usage: $(basename $0) create-restore [--insecure] BACKUP_FILE" ; exit 1 ;;
+  esac
+  if [[ "$1" = "--insecure" ]] ; then
+    insecure=true
+    shift
+  fi
+  BACKUP="$1"
+
+  if command -v k3s &> /dev/null ; then
+    KUBECTL_CMD="k3s kubectl"
+  else
+    check_kubeconfig
+  fi
+
+  if [[ "$1" = "insecure" ]]; then
+    ${KUBECTL_CMD} create -f - <<EOF
+apiVersion: resources.cattle.io/v1
+kind: Restore
+metadata:
+  name: restore-minio
+spec:
+  backupFilename: ${BACKUP}
+  prune: true
+  storageLocation:
+    s3:
+      credentialSecretName: miniocreds
+      credentialSecretNamespace: default
+      bucketName: rancherbackups-insecure
+      endpoint: minio.minio.svc.cluster.local:9000
+      insecureTLSSkipVerify: true
+EOF
+  else
+    ${KUBECTL_CMD} create -f - <<EOF
+apiVersion: resources.cattle.io/v1
+kind: Restore
+metadata:
+  name: restore-minio
+spec:
+  backupFilename: ${BACKUP}
+  prune: true
+  storageLocation:
+    s3:
+      credentialSecretName: miniocreds
+      credentialSecretNamespace: default
+      bucketName: rancherbackups
+      endpoint: minio.minio.svc.cluster.local:9000
+      endpointCA: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUNTVENDQWUrZ0F3SUJBZ0lRV2dVVldDaVpkeU9HcnVOZTZtNGlXakFLQmdncWhrak9QUVFEQWpCTU1Sd3cKR2dZRFZRUUtFeE5EWlhKMFoyVnVJRVJsZG1Wc2IzQnRaVzUwTVN3d0tnWURWUVFMRENObGJHbDVZVzFzWlhaNQpRRVZzYVhsaGJYTXRUVUpRTG1GMGRHeHZZMkZzTG01bGREQWVGdzB5TWpBMU1URXhOREF4TWpCYUZ3MHpNakExCk1URXdNakF4TWpCYU1Fd3hIREFhQmdOVkJBb1RFME5sY25SblpXNGdSR1YyWld4dmNHMWxiblF4TERBcUJnTlYKQkFzTUkyVnNhWGxoYld4bGRubEFSV3hwZVdGdGN5MU5RbEF1WVhSMGJHOWpZV3d1Ym1WME1Ga3dFd1lIS29aSQp6ajBDQVFZSUtvWkl6ajBEQVFjRFFnQUVGSDhVUENsL3ZBSGtNYlRGM0U4eWhTZExOSDJYdWVLVUhucytPNEZSCmhuMDk2T0pLbkdaRmIvSGlXOWlKV2hqNENKNEx1YlN2c2laSlo3WXVEbE05ZmFPQnNqQ0JyekFPQmdOVkhROEIKQWY4RUJBTUNBcVF3RXdZRFZSMGxCQXd3Q2dZSUt3WUJCUVVIQXdFd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZApCZ05WSFE0RUZnUVVJbWpyZXNxbDc4ZkJwd1NWN2xwNGZUNCtObnd3V0FZRFZSMFJCRkV3VDRJRmJXbHVhVytDCkMyMXBibWx2TG0xcGJtbHZnZzl0YVc1cGJ5NXRhVzVwYnk1emRtT0NIVzFwYm1sdkxtMXBibWx2TG5OMll5NWoKYkhWemRHVnlMbXh2WTJGc2dnbHNiMk5oYkdodmMzUXdDZ1lJS29aSXpqMEVBd0lEU0FBd1JRSWdXVDRDVTVpYgpMTmVYSm1oMmxucUV2YWVLZ3FMSFBGZ01PUWcrNFR5Tyt1UUNJUUNJNVdYMUU4NEIrejZ5WDdXS0lCWUpJanRvClJqUWk3NVFuaUYxMHBpMmpLQT09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
```

Comment:
For context, where is this endpoint CA being generated from?

Reply:
There's a setting to a shell variable `PUBLIC_CRT` in `scripts/integration`.

We used to run it through `base64 <-w on Darwin>` but the base64 of that string
was hardcoded into `scripts/deploy: create_backup` by Sebastiaan in June, 2022

Good question. The original crt is still in `scripts/deploy` in `deploy_minio`. Same string as in `scripts/integration`

I'm all for putting the original PUBLIC_CRT in a `scripts/certs` bash script, and then having both `deploy` and `integration` access them via `source $(dirname $0)/certs`

Reply:
No worries, sounds good